### PR TITLE
Set correct owners

### PR DESCRIPTION
**Pull request by Fedosin**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+# See the CONTRIBUTING docs: https://github.com/openshift/origin/blob/master/CONTRIBUTING.adoc
+
+aliases:
+  shiftstack-team:
+    - Fedosin
```
Comment (mandre) [inline OWNERS_ALIASES]: Nit: could you sort alphabetically?

Comment (Fedosin) [reply OWNERS_ALIASES]: Sure! The original order was taken from https://github.com/openshift/cluster-api-provider-openstack/blob/master/OWNERS_ALIASES

Comment (Fedosin) [reply OWNERS_ALIASES]: Done

Comment (pierreprinetti) [reply OWNERS_ALIASES]: > The original order was taken from

ih ih that's me https://github.com/openshift/cluster-api-provider-openstack/commit/ef2821d45f0652f1c34be44511f45de57cc7bad6#diff-0de52d14f35759dfa67139f90c82bd7f

**Pull request merged**